.github/workflows/code_analysis.yml

```diff
@@ -15,38 +15,36 @@ jobs:
                 actions:
                     -
                         name: 'ECS'
-                        run: 'vendor/bin/ecs'
-
+                        run: 'composer run check-cs'
                     -
                         name: 'PHPStan'
-                        run: 'vendor/bin/phpstan'
-
+                        run: 'composer run phpstan'
```

Comment:
Composer adds it's own output wrapper and process wrapper, that add one more level of complexity to debug.
The tool setup is in the configs, so no need to wrap them with composer script. 

The goal of CI is to give as fast and as narrow feedback as possible, so we keep the tool calls here.